crates/iota/src/client_ptb/ptb.rs

```diff
@@ -447,6 +460,10 @@ pub fn ptb_description() -> clap::Command {
             --"preview"
             "Preview the list of PTB transactions instead of executing them."
```

Comment:
Not changing this doc?

Reply:
https://github.com/iotaledger/iota/pull/7881/commits/c0834eaba246b4fca302cc22705f1425bbfb7692